pages/_app.tsx

```diff
@@ -1,6 +1,10 @@
 import "../styles/globals.css";
 import type { AppProps } from "next/app";
+import {Poppins } from '@next/font/google'
 
+const poppins = Poppins({weight: ['300','400','500','600','700','800','900']})
 export default function App({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />;
+  return  <main className={poppins.className}>
+  <Component {...pageProps} />
+</main>;
```

Comment:
Nice work!

Maybe consider the tailwind implementation with next/font?
https://nextjs.org/docs/basic-features/font-optimization#with-tailwind-css